accounts/keystore/passphrase_test.go

```diff
@@ -0,0 +1,240 @@
+package keystore
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/0xPolygon/polygon-edge/helper/common"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	veryLightScryptN = 2
+	veryLightScryptP = 1
+)
+
+func TestKeyStorePassphrase(t *testing.T) {
+	t.Parallel()
+
+	ks := &keyStorePassphrase{veryLightScryptN, veryLightScryptP}
+
+	k1, account, err := storeNewKey(ks, pass)
+	require.NoError(t, err)
+
+	k2, err := ks.GetKey(k1, pass)
+	require.NoError(t, err)
+
+	require.Equal(t, types.StringToAddress(k1.Address), k2.Address)
+
+	require.Equal(t, k2.Address, account.Address)
+}
+
+func TestKeyStorePassphraseDecryptionFail(t *testing.T) {
+	t.Parallel()
+
+	ks := &keyStorePassphrase{veryLightScryptN, veryLightScryptP}
+
+	k1, _, err := storeNewKey(ks, pass)
+	require.NoError(t, err)
+
+	_, err = ks.GetKey(k1, "bar")
+	require.Equal(t, err.Error(), ErrDecrypt.Error())
```

Comment:
IMO this is more readable. It is clear that error is expected here:
```suggestion
	require.EqualError(t, err, ErrDecrypt.Error())
```